src/Faker/Provider/bs_BA/Person.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Faker\Provider\bs_BA;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link http://www.just-think-it.com/sbn/bosnianm.htm
```

Comment:
This source doesn't have a Licence, and I doubt we can reuse the data in a MIT License project.